src/terra-core/TerraCoreDataModel.ttl

```diff
@@ -2412,17 +2458,12 @@ TerraCore:hasSampleTreatmentType
   TerraCore:hasColumnLabel "sample_treatment_type" ;
 .
 TerraCore:hasSampleType
-  a owl:ObjectProperty ;
+  a owl:DatatypeProperty ;
   rdfs:domain TerraCore:Sample ;
   rdfs:label "hasSampleType" ;
+  TerraCore:hasColumnLabel "sample_type" ;
 .
-TerraCore:hasSequenceLocation
-  a owl:ObjectProperty ;
-  rdfs:domain TerraCore:VariantCall ;
-  rdfs:label "hasSequenceLocation" ;
-  rdfs:range TerraCore:SequenceLocation ;
-  skos:prefLabel "hasSequenceLocation" ;
-.
+
```

Comment:
You might want to open and save this in TBC; I believe TBC will move the addition of hasSequenceLocation above (alphabetically where hasLocation was) to here.